### DISCUSSION TITLE
2021/transparently-redirecting-packetsframes-between-interfaces/

### DISCUSSION THREAD

**Discussion by giscus** [hellt/netdevops.me#27] [Show and tell]
# 2021/transparently-redirecting-packetsframes-between-interfaces/

Applying DevOps to networks.

https://netdevops.me/2021/transparently-redirecting-packetsframes-between-interfaces/

**Comment by sipani909** [comment]
I have tried the ovs command 
`ovs-vsctl set bridge $brname other-config:forward-bpdu=true'

as suggested, however I see LACP packets only on tap interface not on the actual bridge or the physical itf

**Comment by BurlyLuo** [comment]
Good Post.  Perfect solutin!
We might as well tell GNS3 and EVENG to call it a day, even if that comes off a little rude. ☺
